deploy/cognito/userpool.tf

```diff
@@ -0,0 +1,38 @@
+variable env {}
+
+resource "aws_cognito_user_pool" "users" {
+  name = "hallebarde-${var.env}"
+
+  admin_create_user_config {
+    allow_admin_create_user_only = true
+  }
+
+  mfa_configuration = "OFF"
```

Comment:
Hm, is it legit Terraform HCL ^^ ? Where is it used ? I don't see any another occurence of it

Reply:
Yeah it is, what bothers you?

Reply:
ping @Mehdi-H

Reply:
1. I don't see where it is used
2. I generally see resource declaration in HCL, I don't think I ever saw simple key values declaration like that. It is not a problem, I just never saw this usage, just curious :d

Reply:
Mmmmh the mfa_configuration line is part of a resource declaration (the userpool)